client/executor/common/src/lib.rs

```diff
@@ -16,8 +16,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! A set of common definitions that are needed for defining execution engines.
-
+#![doc = include_str!("../README.md")]
```

Comment:
Can you please move the docs back here.

Reply:
Ok, I can see I'm too early somehow for this change. Will do.

Reply:
We also don't use this anywhere and I'm not sure that stuff like intra doc links would still work. I honestly would not loose such a good feature for having an external file that some editor can understand better.